pkgs/python-packages/joinmarket/default.nix

```diff
@@ -0,0 +1,119 @@
+{ version
+, src
+, format
+, lib
+
+# TODO: Is it necessary to use this deps check?
+#, buildPythonPackageWithDepsCheck
+, buildPythonPackage
+, pythonOlder
+, pythonAtLeast
+, pythonRelaxDepsHook
+, pytestCheckHook
+, setuptools
+, fetchurl
+, chromalog
+, cryptography
+, service-identity
+, twisted
+, txtorcon
+, python-bitcointx
+, argon2_cffi
+, autobahn
+, bencoderpyx
+, klein
+, mnemonic
+, pyjwt
+, werkzeug
+, libnacl
+, pyopenssl
+}:
+
+#buildPythonPackageWithDepsCheck rec {
```

Comment:
With this line I am getting conflict errors for `pyasn1` and `pyasn1-modules`. That's why I am using the `buildPythonPackage`. Is it ok? I have found [this issue](https://github.com/NixOS/nixpkgs/issues/253131). Is the usage of `buildPythonPackageWithDepsCheck` still necessary?

```
       > INFO: pip is looking at multiple versions of pyasn1-modules to determine which version is compatible with other requirements. This could take a while.
       > ERROR: Cannot install pyasn1-modules==0.3.0 and service-identity==24.1.0 because these package versions have conflicting dependencies.
       > 
       > The conflict is caused by:
       >     service-identity 24.1.0 depends on pyasn1
       >     pyasn1-modules 0.3.0 depends on pyasn1<0.6.0 and >=0.4.6
```

Reply:
Python package deps check issue has been fixed in nixpkgs-unstable, but not in nixpkgs 23.11, so we still need `buildPythonPackageWithDepsCheck`.

The `pyasn1` conflict only seems to happen with nixpkgs-unstable, not with nixpkgs 23.11, which we're using for joinmarket in nix-bitcoin.